integration-cli/docker_cli_attach_unix_test.go

```diff
@@ -120,13 +116,10 @@ func (s *DockerSuite) TestAttachAfterDetach(c *check.C) {
 		c.Fatal("timeout waiting for attach read")
 	}
 
-	if err := cmd.Wait(); err != nil {
-		c.Fatal(err)
-	}
+	err = cmd.Wait()
+	c.Assert(err, checker.IsNil)
```

Comment:
just a random thought, I've seen many of these where can just instead do `c.Assert(cmd.Wait(), checker.IsNil)` but to someone this isn't so readable probably